<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioPlayer</class>
 <widget class="QMainWindow" name="AudioPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qt Media Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="audioplayer.qrc">
    <normaloff>:/imgs/icon.png</normaloff>:/imgs/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: rgb(245, 121, 0);
}</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>861</width>
      <height>563</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="bt_stop">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(85, 87, 83);
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="audioplayer.qrc">
         <normaloff>:/imgs/stop.png</normaloff>:/imgs/stop.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="bt_play">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(85, 87, 83);
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
       <property name="icon">
        <iconset resource="audioplayer.qrc">
         <normaloff>:/imgs/play.png</normaloff>:/imgs/play.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLCDNumber" name="lcd">
       <property name="styleSheet">
        <string notr="true">QLCDNumber{
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="sl_tracking">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="lb_screen">
       <property name="minimumSize">
        <size>
         <width>760</width>
         <height>450</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>760</width>
         <height>450</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="audioplayer.qrc">:/imgs/icon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="sl_volume">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>20</width>
      <height>311</height>
     </rect>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_load">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>494</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 87, 83);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <action name="actionLoad_single_song">
   <property name="text">
    <string>Load single song</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionLoad_Playlist">
   <property name="text">
    <string>Load Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="audioplayer.qrc"/>
 </resources>
 <connections/>
</ui>
